gem/styles/gem-rwanda/auth/ninyampinga.enhanced.scss

```diff
@@ -203,6 +203,7 @@ h1 {
   width: 100%;
   @media (min-width:64em) {
     width: auto;
+    margin: 0 0 0 11.4em;
```

Comment:
@moh-moola Can you please add screenshots of this update.

Reply:
![Screenshot_2019-06-18_11-52-55](https://user-images.githubusercontent.com/5039730/59672472-d469df00-91bf-11e9-834e-c0a6d6be72e6.png)
![Screenshot_2019-06-18_11-53-30](https://user-images.githubusercontent.com/5039730/59672473-d5027580-91bf-11e9-80ba-47a85858deba.png)